test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly follow this [link](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) to re-add this file to the .gitignore